pkg/apis/batch/validation/validation_test.go

```diff
@@ -213,6 +215,20 @@ func TestValidateJob(t *testing.T) {
 			},
 		},
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.TTLAfterFinished) {
```

Comment:
Neither of these test cases are going to run since the feature won't be enable in these tests.